webapp/src/main/java/org/apache/atlas/notification/EntityNotificationListenerV2.java

```diff
@@ -159,14 +145,46 @@ public void onLabelsAdded(AtlasEntity entity, Set<String> labels) throws AtlasBa
 
     private void notifyEntityEvents(List<AtlasEntity> entities, OperationType operationType) throws AtlasBaseException {
         MetricRecorder metric = RequestContext.get().startMetricRecord("entityNotification");
+
+        Map<String,AtlasEntity> differentialEntities  = RequestContext.get().getDifferentialEntitiesMap();
+
+        List<EntityNotificationV2> messages = new ArrayList<>();
+
+        for (AtlasEntity entity : entities) {
+             if (isInternalType(entity.getTypeName())) {
```

Comment:
Was this check present earlier, if not, why do we need it now?

Reply:
Yes that check was present earlier